pkg/binding/bindingpolicy-resolver.go

```diff
@@ -66,26 +65,29 @@ type BindingPolicyResolver interface {
 	// singleton status reporting requirement in the resolution.
 	NoteBindingPolicy(bindingpolicy *v1alpha1.BindingPolicy)
```

Comment:
FYI: Sorry I did not notice this before, but this method makes a BindingPolicyResolver care what those bindingPolicyKey strings are --- this method uses a BindingPolicy's Name as he bindingPolicyKey. So the comment on this interface is inaccurate. Does not have to be fixed in this PR.